packages/wdio-devtools-service/tests/gatherer/trace.test.js

```diff
@@ -217,7 +217,7 @@ test('onLoadEventFired: using min trace time', async () => {
 
     const start = Date.now()
     await traceGatherer.onLoadEventFired()
-    expect(Date.now() - start).toBeGreaterThanOrEqual(299)
+    expect(Date.now() - start).toBeGreaterThanOrEqual(250)
```

Comment:
@christian-bromann according to https://github.com/webdriverio/webdriverio/blob/bc6ff2b374aab716d4f4d4300b82c89f01b5df20/packages/wdio-devtools-service/src/gatherer/trace.js#L156 
`waitForNetworkIdleEvent` and `waitForCPUIdleEvent` are running in parallel which means that overall time should be `100 + 150` instead of `50 + 100 + 150`. Please correct me if I'm wrong

Reply:
added this fix because the test was failing for me with values 294, 298

Reply:
sounds good to me